cooked-validators/tests/Cooked/QuickValueSpec.hs

```diff
@@ -20,7 +21,7 @@ spec = do
 
 customInitialDistribution :: InitialDistribution
 customInitialDistribution =
-  initialDistribution' [(wallet 1, quickValue "goldenCoins" 20)]
+  initialDistribution' [(wallet 1, [Pl.lovelaceValueOf 2000000 <> quickValue "goldenCoins" 20])]
```

Comment:
This seems surprisingly well-tuned :) Shall this be `minAda`?

Reply:
Absolutely! :)